<ui version="4.0" >
 <class>runPIHM</class>
 <widget class="QDialog" name="runPIHM" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PIHM</string>
  </property>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>125</x>
     <y>177</y>
     <width>216</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>220</y>
     <width>538</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>531</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="3" >
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QPushButton" name="runButton" >
      <property name="text" >
       <string> RUN </string>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QPushButton" name="helpButton" >
      <property name="text" >
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>535</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>434</x>
      <y>30</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="browseButton" >
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>16</x>
      <y>40</y>
      <width>102</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select Input Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>16</x>
      <y>90</y>
      <width>102</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Input File ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>132</x>
      <y>38</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="cursorPosition" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFile" >
    <property name="geometry" >
     <rect>
      <x>132</x>
      <y>90</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>runPIHM</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
